apis/clusterresources/v1beta1/exclusionwindow_webhook.go

```diff
@@ -55,6 +55,11 @@ var _ webhook.Validator = &ExclusionWindow{}
 func (r *ExclusionWindow) ValidateCreate() error {
 	exclusionwindowlog.Info("validate create", "name", r.Name)
 
+	if (r.Spec.ClusterID == "" && r.Spec.ClusterRef == nil) ||
+		(r.Spec.ClusterID != "" && r.Spec.ClusterRef != nil) {
+		return fmt.Errorf("only one of dataCenter ID and cluster reference fields should be specified")
```

Comment:
The same here, please change erro message in all such places

Reply:
done